pkg/api/api.go

```diff
@@ -50,6 +50,9 @@ type ConfigHandler func(actualCfg interface{}, defaultCfg interface{}) http.Hand
 type Config struct {
 	SkipLabelNameValidationHeader bool `yaml:"skip_label_name_validation_header_enabled" category:"advanced"`
 
+	// when true the distributor translate OTLP to metadata and forwards it,
```

Comment:
I don't think this comment is necessary, we have the description on the flag to help people understand what this is for.

Reply:
removed